zib/vocabulary/manual/ConceptMap-GeslachtCodelijst-to-AdministrativeGender.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ConceptMap xmlns="http://hl7.org/fhir">
+  <id value="GeslachtCodelijst-to-AdministrativeGender"/>
+  <text>
+    <status value="empty"/>
+    <div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div>
+  </text>
+  <extension url="http://hl7.org/fhir/StructureDefinition/concept-bidirectional">
+    <valueBoolean value="false"/>
+  </extension>
+  <url value="http://nictiz.nl/fhir/ConceptMap/GeslachtCodelijst-to-AdministrativeGender"/>
+  <name value="GeslachtCodelijst_to_AdministrativeGender"/>
+  <title value="GeslachtCodelijst to AdministrativeGender"/>
+  <status value="draft"/>
+  <publisher value="Nictiz"/>
+  <contact>
+    <name value="Nictiz"/>
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="Maps gender codes as found in the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Patient v4.3 (2024)](https://zibs.nl/wiki/Patient-v4.3(2024EN)) and [zib HealthProfessional v4.0 (2024)](https://zibs.nl/wiki/HealthProfessional-v4.0(2024EN)) to &lt;http://hl7.org/fhir/ValueSet/administrative-gender&gt; as found in `Patient.gender` and `Practitioner.gender` in FHIR R4.&#xD;&#xA;&#xD;&#xA;Note: these zibs both define an identical ValueSet with the identical name 'GeslachtCodelijst', but with a different canonical URL. To allow both ValueSets as source for this ConceptMap, no `sourceCanonical` is given."/>
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <targetCanonical value="http://hl7.org/fhir/ValueSet/administrative-gender"/>
+  <group>
+    <!-- 2.16.840.1.113883.5.1 -->
+    <source value="http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender"/>
+    <target value="http://hl7.org/fhir/administrative-gender"/>
+    <element>
+      <code value="M"/>
+      <target>
+        <code value="male"/>
+        <equivalence value="equal"/>
+      </target>
+    </element>
+    <element>
+      <code value="F"/>
+      <target>
+        <code value="female"/>
+        <equivalence value="equal"/>
+      </target>
+    </element>
+    <element>
+      <code value="UN"/>
+      <target>
+        <code value="other"/>
+        <equivalence value="narrower"/>
+        <comment value="Undifferentiated: the gender of a person could not be uniquely defined as male or female, such as hermaphrodite. Other: neither male nor female and not unknown."/>
```

Comment:
Zouden we hier ZIB-1239 moeten verwerken?

Reply:
De omschrijvingen worden niet meegenomen in de conceptmap, alleen de code en die is niet veranderd.